src/components/thumbnailcolumn/thumbnailcolumn.jsx

```diff
@@ -12,7 +12,7 @@ const ThumbnailColumn = props => (
     <FlexRow className={classNames('thumbnail-column', props.className)}>
         {props.items.map((item, key) => {
             const href = `/${props.itemType}/${item.id}/`;
-            if (props.itemType === 'preview') {
+            if (props.itemType === 'projects') {
```

Comment:
very small nit, but this might be better as 'project' rather than 'projects' (the studio-list version is singular 'studio'

Reply:
Oh it needs to be projects for the URLs to work, since it goes to `/itemType/id`, we also use itemType: 'projects' in other places.